sdks/python/apache_beam/transforms/external.py

```diff
@@ -416,7 +416,8 @@ def _normalize(coder_proto):
         unique_name=full_label,
         spec=self._expanded_transform.spec,
         subtransforms=self._expanded_transform.subtransforms,
-        inputs=self._expanded_transform.inputs,
+        inputs={tag: pcoll_renames.get(pcoll, pcoll)
```

Comment:
Was this needed for some specific case ? (just for my information)

Reply:
'inputs' should be also renamed  when there are two or more pcollections. It just worked because the current implementation assumes that original and expanded pcollections are the same if there's only one pcollection.